sources/board.c

```diff
@@ -39,6 +39,8 @@ int add_piece_to_board(unsigned int x, unsigned int y, unsigned int player)
     board.board[y * board.size + x] = player;
     if (player == 1)
         board.turn++;
+    else
+        board.opponent_turn++;
```

Comment:
I supposed that if you know your turn, you know the opponent turn ^^. But never mind.